cardano-api/api/controllers/cardano_tx_controller.go

```diff
@@ -8,8 +8,8 @@ import (
 	"net/http"
 	"time"
 
-	"github.com/Ethernal-Tech/cardano-api/api/model/request"
-	"github.com/Ethernal-Tech/cardano-api/api/model/response"
+	"github.com/Ethernal-Tech/cardano-api/api/model/cardano/request"
```

Comment:
since you've created /model/skyline for skyline models, maybe this should be /model/reactor, instead of /model/cardano